server/constants/constants.go

```diff
@@ -115,14 +118,16 @@ const (
 	WSEventSubscriptionDeleted = "subscription_deleted"
 
 	// Colors
-	ReposIconColor  = "#d74f27"
-	BoardsIconColor = "#53bba1"
+	ReposIconColor    = "#d74f27"
+	BoardsIconColor   = "#53bba1"
+	PipelineIconColor = "#4275E4"
```

Comment:
These colors are present on line 131 as well. I think they are added in two different PRs. We should sync both.